app/frontend/src/components/UserSummary.tsx

```diff
@@ -81,10 +81,25 @@ export default function UserSummary({
       {!user ? (
         <Skeleton />
       ) : (
-        <>{nameOnly ? user.name : `${user.name}, ${user.age}, ${user.city}`}</>
+        <>{nameOnly ? user.name : `${user.name}, ${user.age}`}
+        </>
```

Comment:
I think the same here - you can probably remove the fragment altogether and inline the ternary